errors_CN.md

```
@@ -8,211 +11,353 @@
 
-#### -1021 时间同步问题
+### -1021 时间同步问题
  * 时延过大，服务器根据接请求中的时间戳判定耗时已经超出了recevWindow。请改善网络条件或者增大recevWindow。
  * 时间偏移过大，服务器根据请求中的时间戳判定客户端时间比服务器时间提前了1秒钟以上。(该参数不可由客户端调节)
 
-#### -1022 签名不正确
+### -1022 签名不正确
  * 请求中携带的signature与服务器根据规则计算得到的signature不一致。通常是因为客户端代码中使用的apisecret错误。
 
+### -1033 正在使用的 Comp ID
+ * `SenderCompId(49)` is currently in use. Concurrent use of the same SenderCompId within one account is not allowed.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)